civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -6581,3 +6605,117 @@ function civihr_employee_portal_convert_array_option_value($optionGroup, $fieldN
 
   return $data;
 }
+
+/**
+ * Get an array containing existing configuration labels for specified Report.
+ *
+ * @param string $reportName
+ * @return array
+ */
+function civihr_employee_portal_get_report_configuration_list($reportName) {
```

Comment:
even this method can probably moved to the new file ( in case that was possible ) as I suggested above .